pkgs/tools/misc/zsjed/default.nix

```diff
@@ -0,0 +1,68 @@
+{ coreutils
+, diffutils
+, fetchurl
+, findutils
+, gawk
+, lib
+, makeWrapper
+, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "zsjed";
+  version = "0.3";
+
+  src = fetchurl {
+    url = "https://ytrizja.de/distfiles/zsjed-${version}.tar.gz";
+    sha256 = "1rcrvrqjsv80ldz7cscqbrf98jb0yd2in97irrzlpask699z5k29";
+  };
+
+  nativeBuildInputs = [
+    diffutils
+    makeWrapper
+  ];
+
+  buildInputs = [
+    coreutils # sort
+    findutils
+    gawk
+  ];
+
```

Comment:
```suggestion
```

Reply:
I guess, it's in stdenv, right?

Reply:
Is there a list of packages which are in stdenv?